openslides/users/static/templates/users/group-detail.html

```diff
@@ -0,0 +1,14 @@
+<h1>{{ group.name }}</h1>
+
+<div id="submenu">
+  <a ui-sref="users.group.list" class="btn btn-sm btn-default">
+    <i class="fa fa-angle-double-left fa-lg"></i>
+    <translate>Back to overview
```

Comment:
Interesting. I did not know that!

But you link only says, that optional tags should be omitted, not, what optional tags are. What I can say, optional tags are a completed list of elements specified in the html5 specification

http://www.w3.org/TR/html5/syntax.html (Section 8.1.2.4)

Other tags, especially custom tags can not be optional. So in this case, a ending tag is needed.